auv_control_demos/README.md

```diff
@@ -0,0 +1,321 @@
+      rz/velocity
+      x/velocity
+      y/velocity
+      z/velocity
+    ```
+
+4. Open another terminal and execute the following command:
+
+    ```bash
+    ros2 topic pub /integral_sliding_mode_controller/system_state geometry_msgs/msg/Twist
+    ```
+
+5. In another terminal, execute the following:
+
+    ```bash
+    ros2 run teleop_twist_keyboard teleop_twist_keyboard --ros-args --remap cmd_vel:=/integral_sliding_mode_controller/reference
+    ```
+
+6. If you echo the `/status' topic of any of the controllers, you would be able
+  to see the controller status messages being populated. For example:
+
+    ```bash
+    ros2 topic echo /integral_sliding_mode_controller/status
+    ```
+
+    The output should resemble:
+
+    ```bash
+    ---
+    header:
+      stamp:
+        sec: 1710806114
+        nanosec: 467363470
+      frame_id: ''
+    dof_states:
+    - name: x
+      reference: 0.5
+      feedback: 0.0
+      feedback_dot: 0.0
+      error: 0.5
+      error_dot: 0.0
+      time_step: 0.004412119
+      output: 103.01240834476435
+    - name: y
+      reference: 0.0
+      feedback: 0.0
+      feedback_dot: 0.0
+      error: 0.0
+      error_dot: 0.0
+      time_step: 0.004412119
+      output: 0.0
+    - name: z
+      reference: 0.0
+      feedback: 0.0
+      feedback_dot: 0.0
+      error: 0.0
+      error_dot: 0.0
+      time_step: 0.004412119
+      output: -2.0
+    - name: rx
+      reference: 0.0
+      feedback: 0.0
+      feedback_dot: 0.0
+      error: 0.0
+      error_dot: 0.0
+      time_step: 0.004412119
+      output: 0.0
+    - name: ry
+      reference: 0.0
+      feedback: 0.0
+      feedback_dot: 0.0
+      error: 0.0
+      error_dot: 0.0
+      time_step: 0.004412119
+      output: 0.0
+    - name: rz
+      reference: 0.0
+      feedback: 0.0
+      feedback_dot: 0.0
+      error: 0.0
+      error_dot: 0.0
+      time_step: 0.004412119
+      output: 0.0
+      ```
+
+  This clearly demonstrates that the integral sliding mode controller is
+  functional, as the reference velocity commands sent to the
+  controller are converted to the appropriate wrench values as reflected in the
+  output field.
+
+### Files used for this demo
+
+- Launch Files:
+  - [individual.launch.py](https://github.com/Robotic-Decision-Making-Lab/auv_controllers/blob/main/auv_control_demos/launch/individual.launch.py)
+
+- Controllers:
+  - [Integral Sliding Mode Controller](https://github.com/Robotic-Decision-Making-Lab/auv_controllers/tree/documentation/velocity_controllers)
+
+- Controller Config:
+  - [individual_controller.yaml](https://github.com/Robotic-Decision-Making-Lab/auv_controllers/blob/main/auv_control_demos/config/individual_controller.yaml)
+
+- URDF File:
+  - [individual.urdf.xacro](https://github.com/Robotic-Decision-Making-Lab/auv_controllers/blob/main/auv_control_demos/urdf/individual.urdf.xacro)
+
+## Example 2: Controller Chaining
+
+This example shows how to launch the auv controllers chained together and
```

Comment:
auv -> AUV. Might be good to run a find-all & replace for this

Reply:
okay